### PR TITLE
Versioning Update

### PR DESCRIPTION
**Pull request by jhaag75**
Added versioning folder using new xapi.vocab.pub process

### CODE REVIEW

File: video/v1.2/video.jsonld [hunk 1]
```diff
@@ -0,0 +1,527 @@
...
+  "seeAlso": "https://github.com/liveaspankaj/xapi-video-cop",
+  "author": {
+    "type": "Organization",
+    "name": "Advanced Distributed Learning(ADL) Initiative"
+  },
+  "versions": [
+    {
+      "id": "https://w3id.org/xapi/video/v1.2",
+      "generatedAtTime": "2018-02-05T10:45:00Z",
+      "wasRevisionOf": ["https://w3id.org/xapi/video/v1.1"]
```
Comment (fugu13) [inline video/v1.2/video.jsonld]: This version needs to be included in the `versions` array as well

File: video/v1.2/video.jsonld [hunk 2]
```diff
@@ -0,0 +1,527 @@
...
+          "presence": "included"
+        },
+        {
+          "location": "$.context.extensions['https://w3id.org/xapi/video/extensions/video-playback-size']",
+          "presence": "included"
+        }
+      ]
+    }
+  ],
+  "patterns": [
```
Comment (fugu13) [inline video/v1.2/video.jsonld]: All of these need to be inScheme of the current profile version. That was mistakenly marked optional in the profile spec for Patterns.

File: video/v1.2/video.jsonld (hunk 1 above)
Comment (jhaag75) [reply video/v1.2/video.jsonld]: Thanks @fugu13 !

File: video/v1.2/video.jsonld (hunk 2 above)
Comment (jhaag75) [reply video/v1.2/video.jsonld]: @fugu13 thanks! Will fix and update today!

**Pull request merged**